tests/tests_fabric/strategies/test_grad_clipping.py

```diff
@@ -0,0 +1,125 @@
+from contextlib import nullcontext
```

Comment:
Ok with this, since the tests are already written now. In general, minor suggestion would be to have the grad clipping tested in each strategy specific file, e.g. test/tests_fabric/strategies/xzy etc. This way, when adding an new strategy, one can easily look at the others and know what needs to be tested and to what extent. We could refactor it in a follow up.

Reply:
I can still add this :)